<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeForm</class>
 <widget class="QDialog" name="EmployeeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="employeeNameListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="employeeStatusListWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>141</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addEmployeeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeEmployeeButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>REMOVE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="statusLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
